test/e2e/create_test.go

```diff
@@ -103,19 +102,20 @@ func (tc *testContext) createWindowsMachineSet(replicas int32, windowsLabel bool
 	if err != nil {
 		return nil, err
 	}
-	return machineSet, framework.Global.Client.Create(context.TODO(), machineSet,
-		&framework.CleanupOptions{TestContext: tc.osdkTestCtx,
-			Timeout: cleanupTimeout, RetryInterval: cleanupRetryInterval})
+	return tc.client.MachineClient.MachineSets(clusterinfo.MachineAPINamespace).Create(context.TODO(), machineSet, metav1.CreateOptions{})
```

Comment:
The `client` is repeated in the call sites here. I suggest removing `Client` from `Machine`. Applies to all clients.